pkg/apis/core/v1/conversion.go

```diff
@@ -386,6 +388,63 @@ func Convert_core_PodSpec_To_v1_PodSpec(in *core.PodSpec, out *v1.PodSpec, s con
 	return nil
 }
 
+// The following NodeSpec conversions functions focus on converting
+// NodeSpec.PodCIDR -> NodeSpec.PodCIDRs
+func Convert_core_NodeSpec_To_v1_NodeSpec(in *core.NodeSpec, out *v1.NodeSpec, s conversion.Scope) error {
+	if err := autoConvert_core_NodeSpec_To_v1_NodeSpec(in, out, s); err != nil {
+		return err
+	}
+	if 0 < len(in.PodCIDRs) {
+		out.PodCIDR = in.PodCIDRs[0]
+	}
+	out.PodCIDRs = in.PodCIDRs
+	return nil
+}
+func Convert_v1_NodeSpec_To_core_NodeSpec(in *v1.NodeSpec, out *core.NodeSpec, s conversion.Scope) error {
+	if err := autoConvert_v1_NodeSpec_To_core_NodeSpec(in, out, s); err != nil {
+		return err
+	}
+	// handle cidr/cidrs fields
+	if 0 < len(in.PodCIDR) && 0 < len(in.PodCIDRs) {
```

Comment:
yoda style, sorry

Reply:
083bf4e reverts all remaining yodas to normal mode.